src/main/webapp/app/exercises/shared/exercise-headers/exercise-headers-information/exercise-headers-information.component.html

```diff
@@ -0,0 +1,38 @@
+@if (exercise) {
+    <div class="d-flex flex-row flex-wrap column-gap-3 overflow-auto">
+        @for (informationBoxItem of informationBoxItems; track informationBoxItem) {
+            <jhi-information-box [informationBoxData]="informationBoxItem">
+                @if (informationBoxItem.content.type === 'difficultyLevel') {
```

Comment:
I initially started with only a few conditions and used `if` statements. As the number of conditions grew, I kept adding more `if` statements instead of switching to a `switch case`. However, I've now made the switch to a switch case for better organization. 👍🏼